app/components/application/details/application-details-home.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component } from "@angular/core";
+
+@Component({
+    selector: "bex-application-details-home",
```

Comment:
I really don't liek the `details-home` name we have for when no entities is selected. For node I named it `no-node-selected.component` but not sure this is the right name either